ledger/src/sigverify_shreds.rs

```diff
@@ -304,7 +304,8 @@ fn sign_shred_cpu(keypair: &Keypair, packet: &mut Packet) {
     let sig_start = 0;
     let sig_end = sig_start + size_of::<Signature>();
     let msg_start = sig_end;
-    let signature = keypair.sign_message(&packet.data()[msg_start..]);
+    let message = packet.data(msg_start..).unwrap_or_default();
```

Comment:
How about we remove this unused `sign_shred_cpu` function altogether?

Reply:
yeah, looks like `sign_shred...` methods here are not used.
perhaps this was a wip to support gpu signing for shreds.
I will keep them for now, until there is more evidence if there is value in moving shred signing to gpu or not.